data/transport missions.txt

```diff
@@ -549,6 +555,7 @@ mission "Deep Archaeology 3"
 	
 	on complete
 		payment 200000
+		log `The wood sample from the church I gave to the archaeologist turned out to be very strange, as carbon dating suggests that the church was, at the latest, built around 1700, long before the creation of the hyperdrive.`
```

Comment:
`The wood sample from the church I gave to the archaeologist turned out to be very strange` -> `Gave the wood samples from the church to the archaeologist. They turned out to be very strange, as...`